DevEdu.API/DevEdu.Business.Tests/Data/TopicData.cs

```diff
@@ -0,0 +1,61 @@
+﻿using DevEdu.DAL.Models;
+using System;
+using System.Collections.Generic;
+
+
+namespace DevEdu.Business.Tests.Data
+{
+    public static class TopicData
+    {
+        public const int ExpectedTopicId = 42;
+        public const int TopicId = 1;
```

Comment:
it's a bad practice to have such values as consts
it's better to declare them in each test method separately